docs/user-guide/ze-use-cases.md

```diff
@@ -307,3 +307,70 @@ The following use cases demonstrate the various functionalities available in Zow
    The output displays in the **Output** panel.
 
    ![Issue a TSO command](../images/ze/ZE-TSO-Command.gif)
+
+### Polling a spool file
+
+Users can periodically refresh a spool file during long-running jobs to get the latest job outputs. This avoids having to close and reopen a spool file to get the latest updates.
+
+There are two main ways to poll a spool file &mdash; automatically at set intervals or manually on demand.
+
+#### **Defining a default interval for polling spool files**
+<br/>
+
+1. Click on the **Settings** icon on the **Activity Bar** and select **Settings**.
+2. In either the **User** or **Workspace** tab, click on the **Extensions** option to open the menu.
+3. Select **Zowe Explorer**.
+4. In the **Jobs: Poll Interval** field, enter a valid time interval, in milliseconds.
+ 	- Value must be greater than or equal to 1000 ms (1 second).
+
+#### **Polling a spool file at set intervals**
+<br/>
+
+1. Expand **JOBS** in the **Side Bar**.
+2. Navigate to the spool file by expanding its corresponding profile and job folder.
+3. Right click the spool file and select **Start Polling**.
+    - Repeat this step with additional spool files to poll multiple files simultaneously.
+4. In the **Poll interval (in ms) for: &lt;spoolfilename&gt;** field, the interval value displays.
```

Comment:
For pt. 4, should we explicitly mention that the input box will appear? For example:

"The '**Poll interval (in ms)**' input box is presented before polling begins. Edit or confirm the interval time in the provided field."

If you think this would be more confusing, please feel free to disregard this comment

Reply:
tweaked the wording to this: The **Poll interval (in ms) for: &lt;spoolfilename&gt;** field displays the current interval value.